controllers/storagerequest/storagerequest_controller.go

```diff
@@ -345,6 +353,25 @@ func (r *StorageRequestReconciler) reconcilePhases() (reconcile.Result, error) {
 			r.StorageRequest.Status.Phase = v1alpha1.StorageRequestReady
 		}
 
+		clusterID := util.GetClusterID(r.ctx, r.Client, &r.log)
```

Comment:
Can you please add a comment of this change

Reply:
done.